data/presets/shop/second_hand.json

```diff
@@ -18,5 +18,10 @@
     "tags": {
         "shop": "second_hand"
     },
-    "name": "Consignment/Thrift Store"
+    "name": "Thrift Store",
+    "aliases": [
+        "Thrift Shop",
+        "Consignment Store",
```

Comment:
en_US would be more typically "Consignment shop"

Reply:
Also add "Secondhand store" if not covered by the tag name.  See: https://www.youtube.com/watch?v=l7vRSu_wsNc

Reply:
Secondhand or Second Hand?

Reply:
“Secondhand”, or “Second-Hand” if one’s spell-checker is from a less amalgamated era.